youtube_dl/__init__.py

```diff
@@ -309,6 +313,7 @@ def parse_retries(retries):
 
     ydl_opts = {
         'usenetrc': opts.usenetrc,
+        'usekeyring': opts.usekeyring,
```

Comment:
New options need to be documented in the docstring of ```YoutubeDL``` class.

Reply:
Documented in https://github.com/rg3/youtube-dl/pull/14674/commits/3be4a1db6b6e144390cb337888daba6c4b52c852